examples/stream/stream.cpp

```diff
@@ -407,5 +472,8 @@ int main(int argc, char ** argv) {
     whisper_print_timings(ctx);
     whisper_free(ctx);
 
+    if (wavWriter) {
+        delete wavWriter;
+    }
```

Comment:
```suggestion
```

Reply:
The destructor for an object is automatically called when the object goes out of scope.